src/NServiceBus.Persistence.ServiceFabric.Tests/ComponentTests/Sagas/When_completing_a_saga_with_no_defined_correlation_property.cs

```diff
@@ -17,14 +17,16 @@ public class When_completing_a_saga_with_no_defined_correlation_property : SagaP
         {
             configuration.RequiresFindersSupport();
 
-            var sagaId = Guid.NewGuid();
+            var correlationPropertyData = Guid.NewGuid().ToString();
```

Comment:
If the test is for the no correlation property scenario, wouldn't `correlationPropertyData` be called something else? F.e. `propertyFoundByData`?